beeswax/logs/ad_log.proto

```diff
@@ -34,13 +34,14 @@ message RequestLogMessage {
   optional bid.BidAgentResponse.AgentData agent_data = 5;
 }
 
-// Next Tag: 17
+// Next Tag: 18
 // ImpressionLogMessage is created for every won auction.
 message ImpressionLogMessage {
-  // NOTE: auctionid is an opaque unique id. Client cannot rely on the meaning
-  // of elements within it or the format of the id.
   optional base.EventId auctionid = 1;
+  // NOTE: auction_id_str is an opaque unique id. Clients should not rely on the meaning
+  // of elements within it or on the format of the id.
   optional string auctionid_str = 12;
+  optional string requestid_str = 17;
```

Comment:
Do we want to extend our warning to `requestid_str` as well? I believe the only guarantee we should make on the request id is that it can be used to group all impressions (identified by `auctionid_str` that came in for a single bid request. Also, is this the correct place to actually explain what `requestid` and `auctionid` are (perhaps out of scope for this PR).

Reply:
The internal code for `requestid_str` has a comment very similar to the one for `auctionid_str` that I deliberately left out. I can add the same style comment as we put for `auctionid_str`.

Reply:
I think that would be good. The specific context in our internal proto that I think may be helpful are these two comments:
`  // The unique identifier of the auction per buzz_key.`
`  // The unique identifier of the bid request per buzz_key.`

@hostdp6 is that an accurate representation of these fields in your opinion?

Reply:
Added comments along these lines and per our IRL discussion. Awaiting approval dfrom @hostdp6

Reply:
Yes, this is accurate enough for me.